### PR TITLE
Addresses #4914 : Update Message Setting model and add schema

### PR DESCRIPTION
**Pull request by bhaveshAn**
Addresses #4914 
# Part 1 : Update Message Setting model and add schema

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:

Update Message Setting model and add schema

#### Changes proposed in this pull request:

- Updates MessageSetting Model
-
-

### CODE REVIEW

File: app/models/message_setting.py [hunk 1]
```diff
@@ -1,38 +1,182 @@
...
+        elif action == MAIL_TO_EXPIRED_ORDERS:
+            message = NOTIFS.MAIL_TO_EXPIRED_ORDERS
+        elif action == MONTHLY_PAYMENT_EMAIL:
+            message = NOTIFS.MONTHLY_PAYMENT_EMAIL
+        elif action == MONTHLY_PAYMENT_FOLLOWUP_EMAIL:
+            message = NOTIFS.MONTHLY_PAYMENT_FOLLOWUP_EMAIL
+        elif action == EVENT_IMPORTED:
+            message = NOTIFS.EVENT_IMPORTED
+        elif action == EVENT_IMPORT_FAIL:
+            message = NOTIFS.EVENT_IMPORT_FAIL
```
Comment (srv-twry) [inline app/models/message_setting.py]: In these kind of scenarios, a switch statement is generally preferred.

File: app/models/message_setting.py [hunk 2]
```diff
@@ -1,38 +1,182 @@
...
     def __repr__(self):
         return '<Message Setting %r >' % self.action
 
     def __str__(self):
         return self.__repr__()
 
+    @classmethod
+    def _email_message(self, action):
+        message = {}
+        if action == EVENT_PUBLISH:
```
Comment (maxlorenz) [inline app/models/message_setting.py]: can we abbreviate this? like
```python
if action in [...]:
    message = MAILS[action]
else:
    message = MAILS.__dict__[action]
```

File: app/models/message_setting.py [hunk 3]
```diff
@@ -1,38 +1,182 @@
...
+
+    @hybrid_property
+    def email_message(self):
+        message = self._email_message(self.action)
+        return message
+
+    @classmethod
+    def _notification_message(self, action):
+        message = {}
+        if action == EVENT_PUBLISH:
```
Comment (maxlorenz) [inline app/models/message_setting.py]: Same here

File: populate_db.py [hunk 4]
```diff
@@ -222,9 +222,13 @@ def create_admin_message_settings():
                      "New Session Proposal"]
     for mail in MAILS:
         if mail in default_mails:
-            get_or_create(MessageSettings, action=mail, mail_status=1, notification_status=1, user_control_status=1)
+            get_or_create(MessageSettings, action=mail, mail_status=True,
```
Comment (maxlorenz) [inline populate_db.py]: `mailstatus=1` vs `True` - what does it mean in this context? A comment would be helpful :)

**Pull request merged**